File: Aplicativo_HoraCerta/app/src/main/java/com/example/aplicativo_horacerta/network/PedidoDeConfirmarAlarme.java
package com.example.aplicativo_horacerta.network;

import com.google.gson.annotations.SerializedName;

public class PedidoDeConfirmarAlarme extends ComunicadoJson {

    @SerializedName("idUsuario")
    private String idUsuario;

    @SerializedName("nomeRemedio")
    private String nomeRemedio;

    @SerializedName("dia")
    private String dia;

    @SerializedName("horario")
    private String horario;

    public PedidoDeConfirmarAlarme(String idUsuario, String nomeRemedio, String dia, String horario) {
        super("ConfirmarAlarme");
        this.idUsuario = idUsuario;
        this.nomeRemedio = nomeRemedio;
        this.dia = dia;
        this.horario = horario;
    }

    // Getters e Setters
    public String getIdUsuario() { return idUsuario; }
    public void setIdUsuario(String idUsuario) { this.idUsuario = idUsuario; }
    public String getNomeRemedio() { return nomeRemedio; }
    public void setNomeRemedio(String nomeRemedio) { this.nomeRemedio = nomeRemedio; }
    public String getDia() { return dia; }
    public void setDia(String dia) { this.dia = dia; }
    public String getHorario() { return horario; }
    public void setHorario(String horario) { this.horario = horario; }
}

File: Aplicativo_HoraCerta/app/src/main/java/com/example/aplicativo_horacerta/PedidoDeConfirmarAlarme.kt
package com.example.aplicativo_horacerta.network

import com.google.gson.annotations.SerializedName

data class PedidoDeConfirmarAlarme(
    @SerializedName("idUsuario") val idUsuario: String,
    @SerializedName("nomeRemedio") val nomeRemedio: String,
    @SerializedName("dia") val dia: String,
    @SerializedName("horario") val horario: String
) {
    @SerializedName("operacao")
    val operacao: String = "ConfirmarAlarme"
}